staging/src/k8s.io/client-go/rest/config_test.go

```diff
@@ -373,3 +374,98 @@ func TestCopyConfig(t *testing.T) {
 		}
 	}
 }
+
```

Comment:
I realize that it would break easily, but a test that has a config with every field filled out and the associated hard coded value for `String()` may be useful, if only to know what the "full string" actually looks like.

Reply:
Added to TestConfigSprint